/*
protoc --cpp_out=. --proto_path=. translate.proto
 */

syntax="proto2";
package newtranx.ai_server;

option cc_generic_services = true;

// translate api pb

message TranslateRequest {
    required string text = 1;
    // other arguments ...
};

message TranslateResponse {
    required string result = 1;
    // other arguments ...
};

// following for restful service

message HttpRequest {};
message HttpResponse {};

service RestfulService {
    rpc HandleRequest(HttpRequest) returns (HttpResponse);
};

